documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -328,7 +332,7 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
 
-### <a name="R2064" />R2064	PostOperationIdContainsUrlVerb
+### <a name="R2066" />R2066	PostOperationIdContainsUrlVerb
```

Comment:
It's ok if you'd like to leave the ID to what it currently corresponds in the code, even if it's repeated. In the issue or PR you open you can include a comment to make this change as well.